.env.example

```diff
@@ -1,2 +1,5 @@
-MONGO_URI=<mongo uri>
```

Comment:
Generally trying to avoid non-backwards compatible changes. But I think this change is ultimately more future-proof than just the URI. Being as new as it is, few if anyone is likely using this in prod yet. So it'll work this time around 😄

Reply:
Yes, sadly it's non-backwards compatible, but in my opinion it is a real life quality change. Aside from Node configuration, almost any microservice/webapp/containerized software splits the database configuration, since it is more elastic to changes and scalable, even in production.